specification/storage/resource-manager/Microsoft.Storage/preview/2020-08-01-preview/examples/StorageAccountCreateUserAssignedEncryptionIdentityWithCMK.json

```diff
@@ -0,0 +1,118 @@
+{
```

Comment:
Can the new properties only be set on create account ,but not on update account?

Reply:
As we discussed this is also support in update account, and an update account sample will be add.

Reply:
Example for UpdateAccount has been added in the file - StorageAccountUpdateWithUserAssignedEncryptionIdentityWithCMK.json